recipes/gst-plugins-base/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# gstreamer expects libffi to be in lib64 for some reason.
+mkdir $PREFIX/lib64
```

Comment:
This still feels hackish to me. Don't forget to open an issue in the feedstock so we can try to patch it later.